core_types/src/chain_header.rs

```diff
@@ -24,6 +25,9 @@ pub struct ChainHeader {
     entry_type: EntryType,
     /// Key to the entry of this header
     entry_address: Address,
+    /// Address(es) of the agent(s) that authored and signed this entry.
+    /// Backed by the entry_signature below.
+    sources: Vec<Address>,
     /// agent's cryptographic signature of the entry
     entry_signature: Signature,
```

Comment:
if sources are going to be a Vec, then we should changes this to `entry_signatures` to match right now, otherwise this is too confusing.

Reply:
right, so the idea is that the nth signature was produced by the nth address/key?

Reply:
Are multiple Agents able to create a single entry together?

Reply:
Yes, that's what is supposed to happen in the Holo fuel transactor for example: the two parties exchange the transaction (pre-flight) entry with node-to-node messages and both sign it and both commit it to their source chain.

Reply:
@lucksus no problem. Happy to provide context. And even if HF isn't using it anymore, I figured it would be idiomatic enough that it still needed to show up in core.